V5 Release note - Productivity part (#879)
akeneo/pim-helpcenter: content/updates/productivity/2-Import-export.md

# Easier collaboration with increased import and export capabilities

## Import and export your reference entity records
::: meta-data type="New" ee-only="true" link-to-doc="../articles/import-export-records.html"

Having a hard time initializing the records of your reference entities? We got you covered! You can now import an Excel or CSV file containing your records. We also make it possible for you to export them in Excel or CSV, once enriched. So neat!

> YOu can find this new feature in the `Import` and `Export` menus, under the following job names: 

Reviewer: You

Author: ```suggestion
You can find this new feature in the `Import` and `Export` menus, under the following job names: 
```

- `Reference entity record import in XSLX`,
- `Reference entity record import in CSV`,
- `Reference entity record export in XLSX`,
- `Reference entity record export in CSV`.

![Record export profiles](../img/record-export-profiles.png)

This new feature can also help you bulk update your records or easily duplicate them.

## Export your products with their assets
::: meta-data type="New" ee-only="true" link-to-doc="../articles/exports.html#update-an-export-profile"

It's now possible to export products and their associated assets from the product grid or your product export profiles. Collaboration around product data is more efficient and it will save you time whenever you need to share product information with your internal and external teams.

> 

Reviewer 2: peut-être ajouter le fait que c'est une option ("with media") et que ce n'est pas obligatoire.

Author: ```suggestion

To do so, you can use our new `With media` option when doing a quick export or using your favorite product export profile. Note, that by default, this option is not enabled.
```

![Quick export products with their assets](../img/quick-export-products-with-assets.png)

## Import assets with Excel or CSV files
::: meta-data type="New" ee-only="true" link-to-doc="../articles/import-assets.html"

> Have you ever wanted to mass import assets? It is now possible thanks to this new feature that allows you to import Excel or CSV files containing all your asset values. For sure, it allows you to save a significant amount of time!

Reviewer 2: Peut-être expliquer qu'aujourd'hui, le mass upload ne permet pas de créer en masse des assets dans une famille dont le main media est un media link? (mais c'est peut-être pas une bonne idée hein!)

Author: See my suggestion below.

> Have you ever wanted to mass import assets? It is now possible thanks to this new feature that allows you to import Excel or CSV files containing all your asset values. For sure, it allows you to save a significant amount of time!
> 
> In our `Import` menu, you can now discover a brand new import profile dedicated to this task.

Author: We can be more precise for sure.
```suggestion
To save you a tremendous amount of time, it is now possible to mass import assets via Excel or CSV files containing all your asset values. Indeed, in our `Import` menu, you can now discover a brand new import profile dedicated to this task. 

Note that this feature is only available for assets with media file as main media.
```
